### PR TITLE
Add keepalived-ipfailover image support

### PR DESCRIPTION
**Pull request by knobunc**
Add in the rules needed to build keepalived-ipfailover.

### CODE REVIEW

**Comment by knobunc** [comment]
@stevekuznetsov @bparees PTAL

File: hack/build-local-images.py [hunk 1]
```diff
@@ -156,10 +165,10 @@ def add_to_context(context_dir, source, destination, container_destination):
...
     """
     debug("Adding file:\n\tfrom {}\n\tto {}\n\tincluding in container at {}".format(
-    	source,
-    	join(context_dir, destination),
-		container_destination)
-   	)
+        source,
+        join(context_dir, destination),
+                container_destination)
+        )
```
Comment (bparees) [inline hack/build-local-images.py]: i get scared when i see whitespace changes in python files.

Comment (stevekuznetsov) [inline hack/build-local-images.py]: ack

**Comment by imcsk8** [comment]
Besides the whitespace stuff, code works ok.

File: hack/build-local-images.py [hunk 2]
```diff
@@ -227,4 +236,4 @@ def debug(message):
     print "[ERROR] This script knows how to build:\n\t{}".format(
         "\n\t".join(map(full_name, image_config.keys()))
     )
-    exit(1)
\ No newline at end of file
+    exit(1)
```
Comment (knobunc) [inline hack/build-local-images.py]: Can I have this change?  No trailing newline at the EOF tickles my delicate sensibilities ;-)

Comment (bparees) [inline hack/build-local-images.py]: yes that's not the whitespace i was worried about.  it's the stuff above.  which, upon closer analysis *appears* to be making it consistent w/ the rest of the file?

Comment (stevekuznetsov) [inline hack/build-local-images.py]: Yeah may have been my screw-up ....

Comment (knobunc) [inline hack/build-local-images.py]: It was my editor running the auto-format hook that I run for the gocode (since that format is enforced).  I hadn't really intended to make the change, but when I looked at the diff it made it consistent, so I left it in.  Anyway, the new pull skips that change.  I may submit a separate commit in my next PR to clean the format.
